mlrun/model_monitoring/stream_processing_fs.py

```diff
@@ -584,6 +588,10 @@ def __init__(
 
     def do(self, full_event):
         event = full_event.body
+        logger.info(
```

Comment:
since we are before release of mlrun 1.3.0, it's not good idea to add log per event (we can add this once 1.3.0 is released, if needed)